File: DofusSpa/src/app/_models/harness-recipe.ts
import { Harness } from "./harness";

export interface HarnessRecipe{
    id : number,
    ankamaID : number,
    name : string,
    url : string,
    imgUrl : string,
    quantity : number,
    level : number,
    type: string,
    harnessID : number,
    harness : Harness,
}

File: DofusApi/Models/HarnessRecipe.cs
namespace DofusApi.Models
{
    public class HarnessRecipe
    {
        public int ID { get; set; }
        public int AnkamaID { get; set; }
        public string Name { get; set; }
        public string Url { get; set; }
        public string ImgUrl { get; set; }
        public int Quantity { get; set; }
        public int Level { get; set; }
        public string Type { get; set; }
        public int HarnessID { get; set; }
        public Harness Harness { get; set; }
        
        
    }
}